catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -202,6 +198,17 @@ public CreateResponse create(
     return createResponse;
   }
 
+  private void rollbackStorage(CreateStorageRequest createStorageRequest) {
+    if (createStorageRequest != null) {
+      try {
+        sourceOperations.getStorage().rollback(createStorageRequest);
+      } catch (StorageException e1) {
+        LOGGER.info(
```

Comment:
Should this be logged at the `info` level?

Reply:
I grabbed existing code and pulled it into its own method.